src/addons/conditionwatchers/addonconditionwatcherlocales.cpp

```diff
@@ -39,13 +39,8 @@ AddonConditionWatcherLocales::~AddonConditionWatcherLocales() {
 }
 
 bool AddonConditionWatcherLocales::conditionApplied() const {
-  QString code = SettingsHolder::instance()->languageCode();
-  if (code.isEmpty()) {
-    code = Localizer::systemLanguageCode();
-    if (code.isEmpty()) {
-      code = "en";
-    }
-  }
+  QString code = Localizer::instance()->languageCodeOrSystem();
+  Q_ASSERT(!code.isEmpty());
```

Comment:
What happens if the code is empty and we are in prod?

Reply:
languageCodeOrSystem returns the system language code which cannot be empty. Fallback is 'en'.

Reply:
I ask because of the Q_ASSERT, if the code cannot be empty then it's unnecessary and confusing. Can we remove it?